### ISSUE TITLE
Users should be able to Edit only their own Meetings, Admins all

### ISSUE THREAD

**Issue by ane-gabriela** [xwikisas/application-meeting#63]
Currently in the Meeting Application (Pro) 1.9.8 any user can Edit any meeting.

A suggestion would be to only allow simple users to Edit their own meetings or the meetings they are Leaders of, while Admins can Edit / Remove any of them.

Environment: Windows 10, XWiki 11.3.5 with MySQL 5.7 and Edge 18

![EditAction](https://user-images.githubusercontent.com/40691112/66649466-e6700180-ec36-11e9-9cc4-6626042442d9.jpg)

**Comment by lucaa** [comment]
I disagree with making any rights settings on the meetings by default through the meeting application.
The 'wiki' principle is that this application is a helper for creating structured pages in a wiki which is that it is a **collaborative place** where anyone can contribute, unless otherwise specified for the entities which are wiki pages. 
The "collaboration by default" is very important to me as a feature of XWiki. If the application would be imposing a rights schema for the different roles of the meeting, I would consider that a break of that principle.
It would also encourage rights proliferation, as it would set rights at the level of each page of the meeting.

**Comment by lucaa** [comment]
Regardless of whether it may be a good idea (which I think it isn't), one thing is sure, for me, this is by no means Critical, for the reasons I explained above.
